bin/pycbc_make_skymap

```diff
@@ -224,11 +240,12 @@ def main(trig_time, mass1, mass2, spin1z, spin2z, f_low, f_upper, sample_rate,
         command.append(tmpdir+"/PSD_"+str(rough_time)+"_"+ifo+".txt")
         command.append("--output-file")
         command.append(tmpdir+"/SNRTS_"+str(rough_time)+"_"+ifo+".hdf")
-
+        
```

Comment:
Please revert this change.

Reply:
I seem to have rewritten my previous commits by force-pushing my changes when trying to rebase. 

Even before that, I was unsure what you meant with your comment?

Reply:
I meant to avoid introducing those extra white spaces (there should be no spaces in blank lines).

Reply:
Ah, I see now.
Yes, this is something I am very poor at keeping track of!